modules/docs/src/03-protocols/04-deriving-cli.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_label: Deriving CLIs
+title: Deriving CLIs
+---
+
+- The Smithy4s Decline module provides the capability to derive a [Decline](https://ben.kirw.in/decline/) Cli for your service.
+- Lets revisit our HelloWorld smithy definition from the [Quickstart](modules/docs/src/01-overview/02-quickstart.md)
+
+```kotlin
+namespace smithy4s.hello
```

Comment:
unrelated: maybe we could leverage mdoc to have this example in one place and avoid having multiple copies of it

Reply:
Will deal with all mdoc related stuff after this PR